addons/binding/org.openhab.binding.pixometer/src/main/java/org/openhab/binding/pixometer/internal/config/Annotation.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.pixometer.internal.config;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * The {@link Annotation} class is the representing java model for the json result for a annotation from the pixometer
+ * api
+ *
+ * @author Jerome Luckenbach - Initial Contribution
+ *
+ */
+public class Annotation {
+
+    @SerializedName("id")
```

Comment:
If field name is same as SerializedName this annotation can be omitted.

Reply:
Hey @Hilbrand,

So this has affect on many of my fields in those config classes. (Not only the `id`field.)
Am I right?

Reply:
Yes it affects all fields.

Reply:
Hm i think i got the template for them out a of a gson generator, because i had any better ide and knowledge at the point of writing this.

thanks for clarification anyway. :)